src/app/public/modules/avatar/avatar.component.ts

```diff
@@ -29,51 +29,75 @@ import {
   styleUrls: ['./avatar.component.scss']
 })
 export class SkyAvatarComponent {
-  public get src(): SkyAvatarSrc {
-    return this._src;
-  }
 
+  /**
+   * Indicates whether users can change the image. To select a different image,
+   * users click the image or drag another image on top of it,
+   * much like the `sky-file-drop` component in the
+   * [file attachment module](https://developer.blackbaud.com/skyux/components/fileattachments).
+   */
   @Input()
-  public set src(value: SkyAvatarSrc) {
-    this._src = value;
+  public set canChange(value: boolean) {
+    this._canChange = value;
   }
 
-  public get name(): string {
-    return this._name;
+  public get canChange(): boolean {
+    return this._canChange;
   }
 
+  /**
+   * Specifies the name of the record that the avatar represents.
+   * If the `src` property does not specify an image, the component extracts initials
+   * from the first and last words of this name and displays them in place of the missing image.
+   * To ensure that the component extracts the correct initials, specify a name with no prefix
+   * or suffix. You can specify just the initials with a space between them.
+   */
   @Input()
   public set name(value: string) {
     this._name = value;
   }
 
-  public get canChange(): boolean {
-    return this._canChange;
+  public get name(): string {
+    return this._name;
   }
 
+  /**
+   * Specifies an image to display to identify a record.
+   */
   @Input()
-  public set canChange(value: boolean) {
-    this._canChange = value;
+  public set src(value: SkyAvatarSrc) {
+    this._src = value;
   }
 
+  public get src(): SkyAvatarSrc {
+    return this._src;
+  }
+
+  /**
+   * Specifies the maximum file size for the image in bytes.
+   */
   @Input()
   public maxFileSize = 500000;
 
+  /**
+   * Emits [a `SkyFileItem` object](https://developer.blackbaud.com/skyux/components/avatar#skyfileitem-properties)
+   * when the image is updated.
+   */
```

Comment:
☝️  `SkyFileItem` is part of the `@skyux/forms` library. However, when looking at those docs, they don't include anything about this object yet. Once we migrate forms, this link will need to be updated.

Reply:
Should we remove the link here for now? I assume it's better to remove the link and risk forgetting to add it back in after we migrate forms than to leave a link in place that we know will eventually be a busted link and risk having that broken link in place.

Reply:
Done.